assignments/prototypes.js

```diff
@@ -7,21 +7,43 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
-  
+
 /*
   === GameObject ===
   * createdAt
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function Gameobject(createdAt, dimensions) {
+  this.createdAt = createdAt;
+  this.dimensions = {
+    length: 0,
+    width: 0,
+    height: 0
+  };
+}
+
+Gameobject.prototype.destroy = function() {
+  return `Object was removed from the game.`;
+};
 /*
   === CharacterStats ===
   * healthPoints
   * name
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function Characterstats(healthPoints, name) {
```

Comment:
Looks like we have run into some confusion here. instead of passing in separate attributes, you should pass in one object with all the attributes. That way, when you do your `.call` you can easily spread all of the attributes to inherited keys.